Added 25 companies

RuchiGitHu opened the pull request:
I have added 25 companies listed below :
1. Tower Research
2. Microsoft
3. Cloudera
4. Twitter
5. Yahoo
6. Rubrik
7. Salesforce
8. Slack
9. MindTickle
10. Rippling
11. Walmart
12. Xilinx
13. GitHub
14. Nutanix
15. InMobi
16. NortonLifeLock
17. Cure.Fit
18. Intel
19. Qualcomm 
20. eBay
21. SumoLogic
22. Expedia
23. Paytm
24. Grab
25. VMware.

RuchiGitHu commented:
> I have added 25 companies listed below :
> 
> 1. Tower Research
> 2. Microsoft
> 3. Cloudera
> 4. Twitter
> 5. Yahoo
> 6. Rubrik
> 7. Salesforce
> 8. Slack
> 9. MindTickle
> 10. Rippling
> 11. Walmart
> 12. Xilinx
> 13. GitHub
> 14. Nutanix
> 15. InMobi
> 16. NortonLifeLock
> 17. Cure.Fit
> 18. Intel
> 19. Qualcomm
> 20. eBay
> 21. SumoLogic
> 22. Expedia
> 23. Paytm
> 24. Grab
> 25. VMware.

Please review @riturajjain2000

RuchiGitHu commented:
> I have added 25 companies listed below :
> 
> 1. Tower Research
> 2. Microsoft
> 3. Cloudera
> 4. Twitter
> 5. Yahoo
> 6. Rubrik
> 7. Salesforce
> 8. Slack
> 9. MindTickle
> 10. Rippling
> 11. Walmart
> 12. Xilinx
> 13. GitHub
> 14. Nutanix
> 15. InMobi
> 16. NortonLifeLock
> 17. Cure.Fit
> 18. Intel
> 19. Qualcomm
> 20. eBay
> 21. SumoLogic
> 22. Expedia
> 23. Paytm
> 24. Grab
> 25. VMware.

Fiexed #29

riturajjain2000 reviewed (changes requested):
You made the changes only to json file , add them to readme as well

RuchiGitHu commented:
> You made the changes only to json file , add them to readme as well

Please review @riturajjain2000, I have updated readme and also added one more company which were skipped to my counting, now it has 26 company.

riturajjain2000 reviewed (changes requested):
Resolve conflicts!!

RuchiGitHu commented:
> Resolve conflicts!!

It's Done!!

RuchiGitHu commented:
Why my PR for GSSOC21 labelled issue didn't have any label. Please let me know...
Looking forward for your reply...
